dbms/src/Storages/Page/V3/Universal/UniversalPageIdFormatImpl.h

```diff
@@ -90,7 +90,12 @@ struct UniversalPageIdFormat
         return buff.releaseStr();
     }
 
-    // data is in kv engine, so it is prepend by KV_PREFIX
+    static UniversalPageId toKVStoreKey(UInt64 region_id)
+    {
+        return toFullPageId(toSubPrefix(StorageType::KVStore), region_id);
```

Comment:
There are also a implementation of key format in dbms/src/Storages/Transaction/Keys.h. I think later(not in this PR) we can somehow reorg these codes.

Reply:
It's already done because `dbms/src/Storages/Transaction/Keys.h` is not introduced into master branch.